src/components/MishnaView/SynopsisTable.tsx

```diff
@@ -1,106 +1,131 @@
 }
 
-export default function SynopsisTable(props: Props) {
-  const classes = useStyles()
-  const { synopsis } = props
+const SynopsisTable = (props: Props) => {
+  const classes = useStyles();
+  const { synopsis, line, setManuscriptPopup } = props;
 
   if (!synopsis) {
-    return null
+    return null;
   }
 
-  const sourceFullName = synopsis => {
-      const location = synopsis?.location ? synopsis?.location : ""
-      return `${synopsis?.name} ${location}`
-  }
-  const sourceName = synopsis => {
-    let name = synopsisMap.get(synopsis.id)?.title
+  const sourceFullName = (synopsis) => {
+    const location = synopsis?.location ? synopsis?.location : '';
+    return `${synopsis?.name} ${location}`;
+  };
+  const sourceName = (synopsis) => {
+    let name = synopsisMap.get(synopsis.id)?.title;
     if (!name) {
-      name = synopsis.name
+      name = synopsis.name;
     }
 
-    return name
-  }
-
+    return name;
+  };
 
   return (
     <TableContainer component={Paper}>
       <Table className={classes.table} aria-label="simple table">
         <TableHead></TableHead>
         <TableBody>
-          {synopsis
-            .map((synopsisRow,i) => {
-              const rawText = getSynopsisRaw(synopsisRow);
-              const compositionType = synopsisRow.composition?.composition.type;
-              let className = classes.default;
-              switch(compositionType) {
-                case 'parallel': className = classes.parellel;
+          {synopsis.map((synopsisRow, i: number) => {
+            const rawText = getSynopsisRaw(synopsisRow);
+            const compositionType = synopsisRow.composition?.composition.type;
+            let className = classes.default;
+            switch (compositionType) {
+              case 'parallel':
+                className = classes.parellel;
                 break;
-                case 'excerpt': className = classes.excerpt;
+              case 'excerpt':
+                className = classes.excerpt;
                 break;
-                case 'yalkut': className = classes.excerpt;
+              case 'yalkut':
+                className = classes.excerpt;
                 break;
-                case undefined: className = classes.default;
-              }
-              return (
-                rawText ?
+              case undefined:
+                className = classes.default;
+            }
+            return rawText ? (
               <TableRow key={i}>
-                <Tooltip enterDelay={800} leaveDelay={200}
-                title={sourceFullName(synopsisRow)}>
-                <TableCell 
-                style={{fontWeight:'bold'}}
-                component="td" scope="row">
-                  {sourceName(synopsisRow)}
+                <Tooltip
+                  enterDelay={800}
+                  leaveDelay={200}
+                  title={sourceFullName(synopsisRow)}
+                >
+                  <TableCell
```

Comment:
If it doesn't change the display too much, try adding a Button without styling.
It can be useful at least for accessibility and keyboard navigation with tabs, to move between the sources.